tools/xc7frames2bit.cc

```diff
@@ -165,7 +165,7 @@ BitHeader createBitHeader() {
 
 int writeBitstream(const ConfigurationPackets& packets,
                    const std::string& output_file) {
-	std::ofstream out_file(output_file);
+	std::ofstream out_file(output_file, std::ofstream::out);
```

Comment:
This is still wrong? See https://en.cppreference.com/w/cpp/io/ios_base/openmode

Reply:
Oh, yes. You are right, it should be ``std::ofstream::binary``.
It doesn't change anything in the output file though.